src/Twig/ATIAnalyticsExtension.php

```diff
@@ -67,10 +76,41 @@ public function getATIPublisherAttributes(?string $elementId, ?string $prefix =
         ];
     }
 
+    /**
+     * get the Site parameter for ATI event stats
+     * @see https://confluence.dev.bbc.co.uk/pages/viewpage.action?pageId=175885338#tab--2022260337
+     * @param CoreEntity $coreEntity
+     * @return string
+     */
+    public function getATIDestination($coreEntity)
+    {
+        if (!$this->destVarHelper) {
+            $this->destVarHelper = new DestinationVariableHelper();
+        }
+        return $this->destVarHelper
+            ->getDestinationFromContext($coreEntity, $this->cosmosInfo->getAppEnvironment());
+    }
+
+    /**
+     * get the L2 parameter for ATI event stats
+     * @see https://confluence.dev.bbc.co.uk/pages/viewpage.action?pageId=175885338#tab-79895020
+     * @param CoreEntity $coreEntity
+     * @return string
+     */
+    public function getATIProducer($coreEntity)
```

Comment:
```suggestion
    public function getATIProducer(ProgrammeItem $coreEntity): string
```
segments_list.getContext() returns instance of ProgrammeItem, maybe worth adding this

Reply:
→ 36598b8c